clusterloader2/pkg/measurement/common/apicalllatency.go

```diff
@@ -64,26 +64,26 @@ type apiCallLatencyMeasurement struct{}
 // Execute supports two actions:
 // - reset - Resets latency data on api server side.
 // - gather - Gathers and prints current api server latency data.
-func (*apiCallLatencyMeasurement) Execute(config *measurement.MeasurementConfig) error {
+func (*apiCallLatencyMeasurement) Execute(config *measurement.MeasurementConfig) (measurement.Summary, error) {
```

Comment:
Let's make the return value an array of Summaries instead of a single one - to allow generating multiple summaries within a measurement.

Reply:
done.